src/components/Common/StatementComponent.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import styled from 'styled-components'
+import { Padding } from 'styled-components-spacing'
+import StyledLink from '../styledLink'
+import GreyBackground from '../GreyBG'
+import { Row, Col, Grid } from '../grid'
+import { DisplayTitle } from '../Typography'
+
+const Link = styled(StyledLink)`
+  margin-bottom: 0;
+  margin-left: 0;
+  padding: 0;
+  font-weight: normal;
+  text-decoration: underline;
+  display: initial;
+`
+
+const StatementComponent = ({ text, noPadding }) => (
+  <GreyBackground>
+    <Grid>
+      <Row>
+        <Col width={[1, 1, 1, 10 / 12, 10 / 12, 9 / 12]}>
+          <Padding
+            vertical={{
+              smallPhone: noPadding ? 0 : 3,
+              smallTablet: noPadding ? 0 : 4
+            }}
+          >
+            <DisplayTitle as="h1" textLight>
+              {typeof text === 'string'
```

Comment:
just a sugestion. Wouldn't be better to use the `children` prop instead of the `text` one, when it's just a string?
So it'd become:

```
<DisplayTitle as="h1" textLight>
    { children }
    { text && text.map(....) }
</DisplayTitle>

```

Reply:
and the usage is clearer (I guess)

```
<DisplayTitle>
  some text here
</DisplayTitle>